app/domain/logs.rb

```diff
@@ -489,14 +490,69 @@ module AuthnJwt
         code: "CONJ00096D"
       )
 
-      FOUND_JWT_FIELD_IN_TOKEN = ::Util::TrackableLogMessageClass.new(
+      FoundJwtFieldInToken = ::Util::TrackableLogMessageClass.new(
         msg: "Successfully found field '{0}' in token and its value is '{1}'",
         code: "CONJ00097D"
       )
 
-      FOUND_JWT_IDENTITY= ::Util::TrackableLogMessageClass.new(
+      FoundJwtIdentity= ::Util::TrackableLogMessageClass.new(
         msg: "Successfully found jwt identity '{0}'",
-        code: "CONJ00098D"
+        code: "CONJ00098I"
+      )
+
+      ValidatedJwtStatusConfiguration = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated JWT status configuration",
+        code: "CONJ00099I"
+      )
+
+      ValidatedAccountExists = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated that account exists",
+        code: "CONJ00100D"
+      )
+
+      ValidateAndDecodeTokenPassed = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully passed validate and decode token",
+        code: "CONJ00101D"
+      )
+
+      GetJwtIdentityPassed = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully passed get JWT Identity",
```

Comment:
identity here will always be with a small "i" - no need to capitalize
What does the following mean anyway?, what was successful? What passed? 
```suggestion
        msg: "Successfully passed get JWT identity",
```

Reply:
fixed